contents/handbook/people/hiring-process.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hiring process
+sidebar: Handbook
+showTitle: true
+---
+
+This page will walk you through how we hire at PostHog. The goal is to have a lightweight process that optimizes for speed for the candidate, but above all for quality of hire.
+
+
+## Deciding to hire
+
+Every hire introduces complexity to the organisation and increases our burn.
+
+At the moment, we're not so worried about having a clearly defined job spec. PostHog is changing rapidly, job descriptions are changing rapidly so it's impossible to define a set list of tasks.
+
+See [our strategy](/handbook/strategy/strategy) to find out how many people we should be hiring at any time, and at what pace.
+
+## Advertising the job
+
+There are a couple of places that are worth posting to:
+
+- HackerNews Who's Hiring
+  - Tend to get high quality candidates, and people interested in working at startups. 
+  - See [Tim's comment history](https://news.ycombinator.com/threads?id=timgl) for a template
+- RemoteOK
+  - High volume of candidates, but much lower quality.
+- AngelList
+  - We found Eric through there. Higher quality than RemoteOK and pretty high volume.
+
+## Interviewing
+
+The stages of interviewing are:
+
+- Application
+- Interview with Tim
+- Interview with James
+- Interview with someone from the team
+- 3 days of contracting work
+- Decision
+
+### Application
+
+It's worth reading applications and resume carefully. For current roles, the things we look for are
+
+- Experience with relevant technologies (Python or similar, React or similar, something to do with big data is a bonus)
+- Has started a project from scratch, without outside help
+  - Usually this manifests as having been the founder of a startup, or building an impressive side project. It can also be shown through a big project in the day job, but that requires a bit more digging.
+- Communication. Do they have spelling errors in their cover letter? What does their online presence look like?
+  - More so than other companies, all of our communication is written and public for the world to see. Good written communication is key.
+
```

Comment:
+ a link to the (internal) "CRM" / spreadsheet

Reply:
Done